bats/tests/pyrsia_node_docker_build_service.bats

```diff
@@ -88,6 +97,12 @@ setup() {
   assert $image_exists
 }
 
+@test "No build status is returned when build id does not exist." {
+  local build_id="b024a136-9021-42a1-b8de-c665c94470f4"
```

Comment:
@dasmanas The chance of randomly hitting this  ID in the closed testing env is pretty much 0 (or one to  2^128), the reason why we want to use the valid UID is we want to avoid situation where the ID is rejected because it's not UUID v4, and in this case the test would  fail every time. This test should use UID since we are emulating an environment as close to the real world scenario as possible.